benchmarking.md

```diff
@@ -49,67 +49,76 @@ Results from a benchmark run on MacMini M1.
-593010 records sent, 118602,0 records/sec (115,82 MB/sec), 0,6 ms avg latency, 28,0 ms max latency.
-591993 records sent, 118398,6 records/sec (115,62 MB/sec), 0,6 ms avg latency, 19,0 ms max latency.
-598984 records sent, 119796,8 records/sec (116,99 MB/sec), 0,5 ms avg latency, 22,0 ms max latency.
-580779 records sent, 116155,8 records/sec (113,43 MB/sec), 0,6 ms avg latency, 20,0 ms max latency.
-586869 records sent, 117373,8 records/sec (114,62 MB/sec), 9,0 ms avg latency, 159,0 ms max latency.
-599970 records sent, 119994,0 records/sec (117,18 MB/sec), 2,6 ms avg latency, 86,0 ms max latency.
-602751 records sent, 120550,2 records/sec (117,72 MB/sec), 0,4 ms avg latency, 19,0 ms max latency.
-10000000 records sent, 117096,018735 records/sec (114,35 MB/sec), 1,39 ms avg latency, 323,00 ms max latency, 0 ms 50th, 3 ms 95th, 34 ms 99th, 105 ms 99.9th.
+477187 records sent, 95437,4 records/sec (93,20 MB/sec), 0,4 ms avg latency, 149,0 ms max latency.
+507616 records sent, 101523,2 records/sec (99,14 MB/sec), 0,8 ms avg latency, 54,0 ms max latency.
+506905 records sent, 101381,0 records/sec (99,00 MB/sec), 0,3 ms avg latency, 18,0 ms max latency.
+504944 records sent, 100988,8 records/sec (98,62 MB/sec), 0,7 ms avg latency, 52,0 ms max latency.
+508329 records sent, 101665,8 records/sec (99,28 MB/sec), 0,3 ms avg latency, 18,0 ms max latency.
+507461 records sent, 101492,2 records/sec (99,11 MB/sec), 0,7 ms avg latency, 55,0 ms max latency.
+508605 records sent, 101721,0 records/sec (99,34 MB/sec), 0,3 ms avg latency, 18,0 ms max latency.
+509013 records sent, 101802,6 records/sec (99,42 MB/sec), 1,0 ms avg latency, 61,0 ms max latency.
+502944 records sent, 100588,8 records/sec (98,23 MB/sec), 0,4 ms avg latency, 26,0 ms max latency.
+483576 records sent, 96715,2 records/sec (94,45 MB/sec), 1,1 ms avg latency, 58,0 ms max latency.
+503447 records sent, 100689,4 records/sec (98,33 MB/sec), 0,6 ms avg latency, 42,0 ms max latency.
+508601 records sent, 101720,2 records/sec (99,34 MB/sec), 0,6 ms avg latency, 55,0 ms max latency.
+507329 records sent, 101465,8 records/sec (99,09 MB/sec), 0,3 ms avg latency, 18,0 ms max latency.
+506374 records sent, 101274,8 records/sec (98,90 MB/sec), 1,0 ms avg latency, 56,0 ms max latency.
+504911 records sent, 100982,2 records/sec (98,62 MB/sec), 0,4 ms avg latency, 27,0 ms max latency.
+504096 records sent, 100819,2 records/sec (98,46 MB/sec), 0,7 ms avg latency, 54,0 ms max latency.
+507512 records sent, 101502,4 records/sec (99,12 MB/sec), 0,3 ms avg latency, 18,0 ms max latency.
+505010 records sent, 101002,0 records/sec (98,63 MB/sec), 0,7 ms avg latency, 53,0 ms max latency.
+504117 records sent, 100823,4 records/sec (98,46 MB/sec), 0,3 ms avg latency, 19,0 ms max latency.
+10000000 records sent, 100731,309306 records/sec (98,37 MB/sec), 0,56 ms avg latency, 149,00 ms max latency, 0 ms 50th, 1 ms 95th, 13 ms 99th, 47 ms 99.9th.
 ```
 
 ### Proxy - pass-through
 
```

Comment:
Perhaps a sentence about what pass-through means (I know _we_ know, but eventually other people will be interested).